packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,377 @@
+## User Types API
```

Comment:
We should change the heading level to a single `#`.

Reply:
Done.